stable/minio/values.yaml

```diff
@@ -58,6 +58,7 @@ accessKey: "AKIAIOSFODNN7EXAMPLE"
 secretKey: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
 certsPath: "/etc/minio/certs/"
 configPathmc: "/etc/minio/mc/"
+mcConfigDir: ""
```

Comment:
As suggested `configPathmc` serves the same purpose unfortunately it was never used.